cmd/bb_browser/BUILD.bazel

```diff
@@ -1,5 +1,6 @@
 load("@com_github_buildbarn_bb_storage//tools:container.bzl", "container_push_official")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_layer")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
```

Comment:
Are you sure this load() is needed? You’re not using it, right?

Reply:
Oops, not needed at all. Just the remains of initial experimentation. I removed and rebased.